quantum/backlight/backlight_avr.c

```diff
@@ -121,7 +121,7 @@
 #        define COMxx1 COM1B1
 #        define OCRxx OCR1B
 #    endif
-#elif !defined(B5_AUDIO) && !defined(B6_AUDIO) && !defined(B7_AUDIO)
```

Comment:
these checks do not match the `AUDIO_PIN/AUDIO_PIN_ALT` config

Reply:
oh, good find - thanks!

Reply:
fixed with 0c9301879dae0cc0fd3a887d2c0353e5fd6d9cc3